### PR TITLE
changes for login and registration and incorporate username with validity during the login

### PR DESCRIPTION
**Pull request by saminur**
It now has more attributes for registration, like username and password.
and the login is based on username and password

### CODE REVIEW

**Comment by saminur** [comment]
re

File: src/account_info.py [hunk 1]
```diff
@@ -5,40 +5,69 @@
...
+            return self.data[self.data['ID']==1]
+        else:
+            id_list = sorted(self.data.ID.tolist(), reverse=True)
+            lastID = id_list[0]
+            if name == '':
+                print('Name cannot be empty')
+                return -1
+            elif username in self.data['UserName'].values:
+                print('User Name already in use. Please use another one!')
+                st.error("User Name already in use. Please use another one!")
```
Comment (JesseChen1031) [inline src/account_info.py:45]: This looks like an abbreviation of streamlit and it's causing a compilation error. I think we need to add `import streamlit as st` to this file

**Review by JesseChen1031 — APPROVED** [review]
Nice job as always!

**Comment by saminur** [comment]
committing the changes for username and password for login and registration

**Pull request merged**